### ISSUE TITLE
Delve Blog support

### ISSUE THREAD

**Issue by drdamour** [OpenLiveWriter/OpenLiveWriter#839]
Anyone try to get it working with the newish delve blogs in sharepoint / office 365? known issues?

**Comment by nvella** [comment]
I have little experience with SharePoint myself, but it seems Delve blogs are a separate thing altogether from the regular SharePoint blogs which we currently support. One option would be to implement a client using the 365 .NET Graph APIs, but it seems Microsoft hasn't opened many APIs yet for Delve.

**Comment by drdamour** [comment]
Yeah and it seems there might be two iterations I’d delve blogs... one from ~2016 that showed up at /portals/personal and a newer iteration that shows up in /portals/hub the former is accessed via _api by example at https://www.c-sharpcorner.com/article/retrieve-office-365-delve-blogs-programmatically-using-rest-api/ while the latter is described a bit at the end of https://sharegate.com/blog/office-365-delve-blogs-explained. 

Even then I’m not convinced they aren’t the same...seems some reason some people’s delve blogs appear at different addresses. Beyond that it’s unclear if just creating the file will create a blog or you have to go thru a specific service...I’ll try messing with that but I haven’t used the share point api in 7 years...

My biggest concern was the weird format of the json  file they mention, is it easy for writers providers to produce a structure like that?

**Comment by nvella** [comment]
It would be doable, but a challenge when it covers implementing all the rich content OLW offers users to insert. Writer authors HTML from top to bottom, so a kind of conversion algorithm would be required to transform it into the JSON format used by Delve; some content like embeds or complex custom code would have to be omitted. However, it certainly would be possible to implement a client for basic post authoring.

**Comment by drdamour** [comment]
Yeah just using delve it’s got all types of buggy spa issues where data is lost when multiple tabs are open and json escaping goes crazy.  My tab became \\\\\\\\\\\\\\\\t after a few saves.

**Comment by drdamour** [comment]
Sniffing traffic it seems like most the bugs are in the delve GUI in constructing the json...so it’ll probably work just if you ever had a post editing in web and writer at the same time all hell will break loose

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
